requirements.txt

```diff
@@ -1,7 +1,7 @@
 # For all tests.
 ddt
 fauxfactory
-nailgun~=0.8.0
+nailgun>=0.8.0
```

Comment:
Could we also specify less than 0.9.0? Something like: `nailgun>=0.8.0,<0.9.0`? I have no idea if that syntax is correct, by the way. But something similar is possible.